lib/car.rb

```diff
@@ -0,0 +1,19 @@
+class Car
+  attr_accessor :monthly_payment, :loan_length, :make, :make_then_model, :model, :color
```

Comment:
You wouldn't need to add `make_then_model` to your `attr_reader` because we only ever need access to them separately.